extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -165,6 +186,40 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
       addExperimentIdToElement(verifierEid, this.element);
     }
 
+    const correlator = getCorrelator(this.win);
+    this.ampAnalyticsPageLoadMetricsConfig_ = /** @type {JsonObject}*/ ({
```

Comment:
I'm tempted to move this to ./utils.js...opinions?

Reply:
Nit: you can use dict() to make a JsonObject without casting.